doc/source/release/1.15.0-notes.rst

```diff
@@ -206,6 +208,14 @@ and support several Array API compatible array libraries in addition to NumPy
 
 `scipy.special` improvements
 ============================
+- New functions offering improved Legendre function implementations with a
+  more consistent interface. See respective docstrings for more information.
+
+  - `scipy.special.legendre_p`, `scipy.special.legendre_p_all`
+  - `scipy.special.assoc_legendre_p`, `scipy.special.assoc_legendre_p_all`
+  - `sph_harm_y`, `sph_harm_y_all`
+  - `sph_legendre_p`, `sph_legendre_p_all`,
```

Comment:
For the last two bullets,  could you either use double backticks or resolve their full namespaces? At the moment, the relnotes render shows italic instead of the usual way we'd display it:

![image](https://github.com/user-attachments/assets/4ce182cb-b6fb-4670-a32b-4f631f468ca7)